amoro-ams/pom.xml

```diff
@@ -408,6 +409,66 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.swagger.core.v3</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>2.2.10</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Do we need the dependency by default?
Or should we separate them for the doc and the SDK?